Packages/Tracking/OpenXR/Runtime/Scripts/OpenXRLeapProvider.cs

```diff
@@ -51,18 +51,34 @@ public Camera mainCamera
             }
         }
 
-        public override bool CanProvideData { get { return CheckOpenXRAvailable(); } }
+        public override TrackingSource TrackingDataSource { get { return CheckOpenXRAvailable(); } }
 
-        private bool CheckOpenXRAvailable()
+        private TrackingSource CheckOpenXRAvailable()
         {
+            if (_trackingSource != TrackingSource.NONE)
+            {
+                return _trackingSource;
+            }
+
             if (XRGeneralSettings.Instance.Manager.activeLoader.name == "Open XR Loader" &&
-                OpenXRSettings.Instance.GetFeature<HandTrackingFeature>() != null &&
-                OpenXRSettings.Instance.GetFeature<HandTrackingFeature>().SupportsHandTracking)
+            OpenXRSettings.Instance.GetFeature<HandTrackingFeature>() != null &&
+            OpenXRSettings.Instance.GetFeature<HandTrackingFeature>().SupportsHandTracking)
```

Comment:
So, I have a question about expectations. I've tested the various pathways and debugged the code each time. When I uninstall our SDK, but continue to have OpenXR hand tracking enabled SupportsHandTracking comes back as true (but not for ours). I assume that it's picking up hand tracking for support from somewhere else. Anyway, I'll check with Rob/Alex on this.

Reply:
Yeah this will be via SteamVR or Oculus, Steam in particular will tell you you have OpenXR hand tracking all the time as it emulated hand tracking with controllers where possible